.github/workflows/lakmus-release.yml

```
@@ -0,0 +1,80 @@
+name: Scan/Release Lakmus Image and Build Kustomize OCI Artifact
+
+# env:
+#   FLUX_ARTIFACT_NAME: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)